config/samples/all-in-one/ita-kbs-config.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kbs-config
```

Comment:
A ConfigMap with the same name already exists in the directory. This causes problems when running commands like `kustomize build .`.
The same applies for the resource-policy.
What about changing the object name for ita specific resources?
Please check the microservices folder as well.

Reply:
I've changed the name to ita-kbs-config, and did the same for the resource-policy.